Source/Core/AudioCommon/OpenALStream.cpp

```diff
@@ -86,13 +86,13 @@ void OpenALStream::Stop()
 
   thread.join();
 
-  alSourceStop(uiSource);
-  alSourcei(uiSource, AL_BUFFER, 0);
+  alSourceStop(ui_source);
```

Comment:
This commit needs to be split into "variable name changes" and "non-variable name changes." It's difficult to tell logic changes apart from syntactical cleanups at a glance.